.github/workflows/ros-workflow.yml

```diff
@@ -101,7 +101,7 @@ jobs:
       run: mobros raise --workspace="$(pwd)"
 
```

Comment:
check line 93.

bump to your [1.0.1.23](https://github.com/MOV-AI/mobros-build-system/releases/tag/1.0.1.23)

check line 53,

bump to your https://github.com/MOV-AI/containers-ros-buildtools/releases/tag/v1.1.7

Then to test your changes in a ros repos, simply change the 3 yml files to have
uses: MOV-AI/.github/.github/workflows/ros-workflow.yml@feature/dp-945
instead of 
uses: MOV-AI/.github/.github/workflows/ros-workflow.yml@v1